pkg/integrations/kafka_lag_exporter/kafka_lag_exporter.go

```diff
@@ -0,0 +1,173 @@
+package kafka_lag_exporter
+
+import (
+	"fmt"
+
+	"github.com/go-kit/kit/log"
+	"github.com/grafana/agent/pkg/integrations"
+	"github.com/grafana/agent/pkg/integrations/config"
+
+	"github.com/Shopify/sarama"
+	kafka_exporter "github.com/davidmparrott/kafka_exporter/v2/exporter"
+)
+
+// DefaultConfig holds the default settings for the elasticsearch_exporter
+// integration.
+var DefaultConfig = Config{
+	KafkaUri:                []string{"localhost:11091"},
+	UseSASL:                 true,
+	UseTLS:                  false,
+	SaslUsername:            "test",
+	SaslPassword:            "test",
+	KafkaVersion:            sarama.V2_0_0_0.String(),
+	UseZooKeeperLag:         false,
+	Labels:                  "dev-cluster",
+	MetadataRefreshInterval: "1m",
+	AllowConcurrent:         true,
+	MaxOffsets:              1000,
+	PruneIntervalSeconds:    30,
+}
+
+type Config struct {
+	Common config.Common `yaml:",inline"`
+
+	// Exporter configuration
+
+	//Address array (host:port) of Kafka server
+	KafkaUri []string `yaml:"kafka_uris,omitempty"`
+
+	//Connect using SASL/PLAIN
+	UseSASL bool `yaml:"use_sasl,omitempty"`
+
+	//Only set this to false if using a non-Kafka SASL proxy
+	UseSASLHandshake bool `yaml:"use_sasl_handshake,omitempty"`
+
+	//SASL user name
+	SaslUsername string `yaml:"sasl_username,omitempty"`
+
+	//SASL user password
+	SaslPassword string `yaml:"sasl_password,omitempty"`
+
+	//The SASL SCRAM SHA algorithm sha256 or sha512 as mechanism
+	SaslMechanism string `yaml:"sasl_mechanism,omitempty"`
+
+	//Connect using TLS
+	UseTLS bool `yaml:"use_tls,omitempty"`
+
+	//The optional certificate authority file for TLS client authentication
+	TlsCAFile string `yaml:"tls_cafile,omitempty"`
+
+	//The optional certificate file for TLS client authentication
+	TlsCertFile string `yaml:"tls_certfile,omitempty"`
+
+	//Password for the certificate file for TLS client authentication
+	TlsCertFilePassword string `yaml:"tls_certfile_password,omitempty"`
+
+	//The optional key file for TLS client authentication
+	TlsKeyFile string `yaml:"tls_keyfile,omitempty"`
+
+	//Password for the key file for TLS client authentication
+	TlsKeyFilePassword string `yaml:"tls_keyfile_password,omitempty"`
+
+	//If true, the server's certificate will not be checked for validity. This will make your HTTPS connections insecure
+	TlsInsecureSkipTLSVerify bool `yaml:"tls_insecure_skip_tlsverify,omitempty"`
+
```

Comment:
Can you rename these to match up with Go naming conventions? i.e., acronyms should be capitalized, so `TLSCAFile` (or just `CAFile` since it implies TLS) and `SASLUsername` etc above